src/networking.md

```diff
@@ -17,6 +17,12 @@ blockchain](./blockchain-runtime-interface.md#runtime-functions) to connect to m
 to the [mixnet topology](./topology.md). If no external addresses have been published for a
 mixnode, or none of them work, nodes should attempt to discover addresses using the libp2p DHT.
 
+// Here I am not sure, do we have the Dht? or if we are talking about discovery dht, we should name
```

Comment:
I'm just talking about the libp2p peer ID --> address DHT. At least I thought that was a thing, and that Substrate would use it if you just asked to connect to a peer ID without providing any addresses? Maybe I'm wrong about this though.

Reply:
> I'm just talking about the libp2p peer ID --> address DHT

that is the one I am calling the discovery dht, I think it is a substrate specific one.